index.js

```diff
@@ -85,6 +95,11 @@ const buildDiffImage = (png1, png2, options, callback) => {
         const color1 = png1.getPixel(x, y);
         const color2 = png2.getPixel(x, y);
 
+        if (isIgnored) {
+            result.setPixel(x, y, blend(ignoredOverlay, blend(color1, color2, 0.5), 0.5));
```

Comment:
It's normal blend of 50% highlightColor, 25% color1 and 25% color2.
It should contain both color1 and color2, because inside ignore area both colors are equaly important.
And it should also differ from surroundings, that's why I added highlightColor.

Reply:
This is a very specific and not expected case. So anything related to this should be removed.

Reply:
@sipayRT need your comment here